src/components/Modals/RemoveLiquidity/index.js

```diff
@@ -0,0 +1,196 @@
+import React, { useEffect, useState } from 'react';
+import PropsTypes from 'prop-types';
+import { BN, BN_HUNDRED, BN_MILLION } from '@polkadot/util';
+import { useDispatch, useSelector } from 'react-redux';
+import { useForm } from 'react-hook-form';
+import ModalRoot from '../ModalRoot';
+import stylesModal from '../styles.module.scss';
+import Button from '../../Button/Button';
+import { calculatePooled, getDecimalsForAsset } from '../../../utils/dexFormatter';
+import { AssetsPropTypes, ReservedAssetPropTypes } from '../../Wallet/Exchange/proptypes';
+import ProgressBar from './ProgressBar';
+import styles from './styles.module.scss';
+import { dexActions } from '../../../redux/actions';
+import { blockchainSelectors, dexSelectors } from '../../../redux/selectors';
+import { calculateAmountMin, formatAssets } from '../../../utils/walletHelpers';
+
+const listPercent = [25, 50, 75, 100];
+
+function RemoveLiquidityModal({
+  handleModal,
+  assets,
+  reserved,
+  lpTokensBalance,
+  liquidity,
+}) {
+  const {
+    handleSubmit,
+    register,
+  } = useForm({
+    mode: 'onChange',
+  });
+  const dispatch = useDispatch();
+  const userWalletAddress = useSelector(blockchainSelectors.userWalletAddressSelector);
+  const withdrawalFee = useSelector(dexSelectors.selectorWithdrawalFee);
+  const {
+    asset1,
+    asset2,
+    assetData1,
+    assetData2,
+    asset1ToShow,
+    asset2ToShow,
+  } = assets;
+  const [asset1Amount, setAsset1Amount] = useState(0);
+  const [asset2Amount, setAsset2Amount] = useState(0);
+  const [percentBurnTokens, setPercentBurnTokens] = useState(0);
+  const [tokensToBurnState, setTokensToBurnState] = useState(null);
+  const decimals1 = getDecimalsForAsset(asset1, assetData1?.decimals);
+  const decimals2 = getDecimalsForAsset(asset2, assetData2?.decimals);
+
+  const calculateAssetToBurn = (numberValue) => {
+    const tokensToBurn = new BN(lpTokensBalance).mul(new BN(numberValue)).div(BN_HUNDRED);
+    setTokensToBurnState(tokensToBurn);
+    const fee = new BN(withdrawalFee);
+    const calculatedAmount1 = calculatePooled(tokensToBurn, liquidity, reserved.asset1);
+    const calculatedAmount2 = calculatePooled(tokensToBurn, liquidity, reserved.asset2);
+    const asset1Data = !fee.isZero()
+      ? calculatedAmount1.sub(calculatedAmount1.mul(fee).div(BN_MILLION))
+      : calculatedAmount1;
+    const asset2Data = !fee.isZero()
+      ? calculatedAmount2.sub(calculatedAmount2.mul(fee).div(BN_MILLION))
+      : calculatedAmount2;
```

Comment:
```suggestion
    const asset1Data = calculatedAmount1.sub(calculatedAmount1.mul(fee).div(BN_MILLION));
    const asset2Data = calculatedAmount2.sub(calculatedAmount2.mul(fee).div(BN_MILLION));
```
Multiplication will handle `fee = 0` just fine